### PR TITLE
Do not load external bootstrap resources for cdn_resources = 'in_line' | 'local'

### PR DESCRIPTION
**Pull request by FieteO**
The `cdn_resources` option should control how external resources and stylesheets should be loaded.
Currently bootstrap is being fetched from CDN irrespective of the `cdn_resources` option.

Also I have observed some inconsistencies in the `local` option where the `vis-network` css and js files are not being taken from the `lib` folder but rather externally fetched. Please correct me here if my understanding of how these options are intended to work is wrong.

To summarise the changes:
- fetch bootstrap from CDN when `cdn_resources = 'remote'`
- use resources from `lib/` when `cdn_resources = 'local'`
- remove the double heading (which is also being fixed in #191)
- run `example.ipynb` to update the samples

### CODE REVIEW

**Review by BerserkerGaruk — COMMENTED** [review]
Went through the commit, I agree with the changes made, but I'm going to need to verify the hash between the bootstrap and visj min.js and .min.css files on my side before merging.

**Comment by FieteO** [comment]
Do you need anything from my side for validating the checksums? @BerserkerGaruk

**Comment by FieteO** [comment]
@BerserkerGaruk @summerswallow-whi , I have rebased the branch to resolve the merge conflicts that occurred in the meantime. I would appreciate it, if you could take a look, since the MR is open for a couple of month now

**Comment by fohrloop** [comment]
Is there anything that could be done to help with this?

**Review by wey-gu — APPROVED** [review]
Thanks a lot for this fix!

**Comment by wey-gu** [comment]
I had verified e2e this change on my offline code with `in_line`, it worked like a charm

cc @summerswallow-whi @BerserkerGaruk 

Thanks!
